generic.html

```diff
@@ -31,7 +31,10 @@ <h1><a href="index.html">Jasyn Marais</a></h1>
 							<h2>Menu</h2>
 							<ul class="links">
 								<li><a href="index.html">Home</a></li>
-								<li><a href="generic.html">About</a></li>
+								<li><a href="generic.html">Now</a></li>
+								<li><a href="https://github.com/jasynmarais">Github</a></li>
+								<li><a href="https://www.linkedin.com/in/jasynmarais/">LinkedIn</a></li>
+								<li><a href="https://twitter.com/jasynmarais">Twitter</a></li>
```

Comment:
You could install [prettier - Coder formatter](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) to help format your code to be more readable